TransformationSystem/Agent/TransformationCleaningAgent.py

```diff
@@ -190,6 +193,63 @@ def execute(self):
       self.log.error("Could not get the transformations", res['Message'])
     return S_OK()
 
+  def finalize(self):
+    """ Only at finalization: will clean ancient transformations (remnants)
+        1) get the transformation IDs of jobs that are older than 1 year
+        2) find the status of those transformations. Those "Cleaned" and "Archived" will be
+        cleaned and archived (again)
```

Comment:
A few characters indent to keep this as part of the list (I think) (Not sure about exact number of spaces)
```suggestion
            cleaned and archived (again)
```